src/app/test/helpers/page-route.ts

```diff
@@ -0,0 +1,40 @@
+import { Type } from "@angular/core";
+import { RouterStateSnapshot } from "@angular/router";
+import { getPageInfos, IPageComponent } from "@helpers/page/pageComponent";
+import { PageInfo } from "@helpers/page/pageInfo";
+
+export function assertPageInfo<T>(
```

Comment:
document this.

Something subtle I just noticed is you're only testing these routes at their top level.

E.g. for harvest details page your test is essentially:

`test title is "test name"`

and not

`test title is "Ecosounds | Projects | Samford | Uploads | test name"`.

The approach you've taken is you don't need to test every segment in the title if you have tests for every title in the parent hierarchy by itself.
However, in theory a title generator can use any of the route data available to it to generate a title. So that is never tested currently. 

At the minimum document about these limitations and how these tests do not reflect actual generation of titles.

Reply:
There's an assertion that the correct route data and page segments are used to construct the route title [here](https://github.com/QutEcoacoustics/workbench-client/pull/2034/files/38f8dbeb1411fd4e81459797451ef77324a634a1#diff-002e1a20416b0120fbe06390465dd02fb2d00c771c5ae171766211daef759308R244-R270)

However, this test does not cover pages with multiple mounting points e.g. sites / point details component, I'll add another assertion to this tests helper method to assert the correct title is used at the different mounting points.